internal/testutil/acc/name.go

```diff
@@ -40,17 +37,3 @@ func RandomIP(a, b, c byte) string {
 func RandomEmail() string {
 	return fmt.Sprintf("%s-%s@mongodb.com", prefixName, acctest.RandString(10))
 }
-
-func ProjectIDGlobal(tb testing.TB) string {
-	tb.Helper()
-	return projectID(tb, prefixProjectKeep+"-global")
-}
-
-func projectID(tb testing.TB, name string) string {
-	tb.Helper()
-	SkipInUnitTest(tb)
-	resp, _, _ := ConnV2().ProjectsApi.GetProjectByName(context.Background(), name).Execute()
-	id := resp.GetId()
-	require.NotEmpty(tb, id, "Project name not found: %s", name)
-	return id
-}
```

Comment:
because I realized that projects can't have more than 25 clusters so I don't think that's an option any more, at least for generic projects that don't need any special configuration.
for projects than need some setup (like triggers) we'll probably have a global project for triggers.

but in the resource in this PR it doesn't need any special config so i think it's better not to use a global project for it (it can use a global project for that resource, but not shared by all/many resources)